rel-eng/tito.props

```diff
@@ -921,6 +921,7 @@ whitelist = nodejs-babel-plugin-module-resolver
   rubygem-foreman_setup
   rubygem-foreman_snapshot_management
   rubygem-foreman_spacewalk
+  rubygem-foreman_statistics
```

Comment:
could you add the same to the `foreman-plugins-nightly-el8` section too please?

Reply:
your wish is my command :upside_down_face: